openslides/agenda/static/templates/agenda/item-list.html

```diff
@@ -34,7 +34,7 @@ <h1 translate>Agenda</h1>
         <button os-perms-lite="agenda.can_manage" class="btn btn-default"
             ng-click="isDeleteMode = !isDeleteMode; uncheckAll()">
           <i class="fa fa-check-square-o"></i>
-          <translate>Select</translate>...
+          <translate>Select</translate> ...
```

Comment:
The dots should be inside the translation block. Maybe there is a language where they do not add the whitespace or use something else then the dots.